corehq/apps/hqadmin/management/commands/cchq_prbac_bootstrap.py

```diff
@@ -170,16 +170,22 @@ def remove_grant(self, priv_slug, dry_run=False):
 
     BOOTSTRAP_PLANS = [
         Role(slug='community_plan_v0', name='Community Plan', description=''),
+        Role(slug='community_plan_v1', name='Community Plan', description=''),
         Role(slug='standard_plan_v0', name='Standard Plan', description=''),
         Role(slug='pro_plan_v0', name='Pro Plan', description=''),
         Role(slug='advanced_plan_v0', name='Advanced Plan', description=''),
         Role(slug='enterprise_plan_v0', name='Enterprise Plan', description=''),
     ]
 
-    community_plan_features = [
+    community_plan_v0_features = [
+        privileges.EXCEL_DASHBOARD,
```

Comment:
They are. I added the privs separately to `standard_plan_features`, although it might make more sense to instead do
```
standard_plan_features = community_plan_v0_features + [ ... ]
```